### DISCUSSION TITLE
New arch specific jdownloader binaries failing to install

### DISCUSSION THREAD

**Discussion by p-linnane** [Homebrew/discussions#3414] [Casks]
Upstream is providing new arch specific binaries for the jdownloader installer. I attempted to update the cask, but installation fails with the below output. 

`Error: Failure while executing; '/usr/local/Caskroom/jdownloader/46316/JDownloader\ 2\ Installer.app/Contents/MacOS/JavaApplicationStub -dir /Applications -q -Dinstall4j.suppressStdout=true -Dinstall4j.debug=false -VcreateDesktopLinkAction\$Boolean=false -VaddToDockAction\$Boolean=false' exited with 22.`

Manually running the command in question outside of `brew` works just fine. Below is a gist of the updated cask. Any help here would be appreciated.

https://gist.github.com/p-linnane/de804fd31b4941670529f4f96c567fef

**Comment by SMillerDev** [comment]
Can you make a pull request so CI can test it and report the errors?

**Reply by p-linnane** [reply]
Just reopened [#126673](https://github.com/Homebrew/homebrew-cask/pull/126673)
